mdoc/src/main/scala/mdoc/internal/worksheets/WorksheetProvider.scala

```diff
@@ -54,8 +52,8 @@ class WorksheetProvider(settings: Settings) {
 
     EvaluatedWorksheet(
       reporter.diagnostics.map(d => d: i.Diagnostic).toSeq.asJava,
-      decorations.toIterator
```

Comment:
Why drop “toIterator”?

Reply:
Because `decorations` is already an Iterator